### PR TITLE
setParameters(), replaceTrack(), & insertDTMF() reject after stop().

### PR DESCRIPTION
**Pull request by jan-ivar**
Fixes https://github.com/w3c/webrtc-pc/issues/2820.

***
<a href="https://pr-preview.s3.amazonaws.com/jan-ivar/webrtc-pc/pull/2829.html" title="Last updated on Apr 3, 2023, 9:01 PM UTC (d7b7183)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-pc/2829/c3e196e...jan-ivar:d7b7183.html" title="Last updated on Apr 3, 2023, 9:01 PM UTC (d7b7183)">Diff</a>

### CODE REVIEW

**Comment by jan-ivar** [comment]
[WPT](https://github.com/web-platform-tests/wpt/blob/c29a0343e99749d4607cb7e6ac64a8de1538288e/webrtc/RTCRtpTransceiver.https.html#L1232) already checks for this, so only a spec-text update is needed to match.

**Comment by alvestrand** [comment]
This aligns with reasonable behavior (reject when "stop" has been called) and implementations. Seems to be a reasonable thing to merge.

File: amendments.json [hunk 1]
```diff
@@ -280,6 +280,13 @@
       "type": "correction",
       "status": "candidate",
       "id": 24
+    },
+    {
+      "description": "Reject setParameters(), replaceTrack(), & insertDTMF() after stop()",
+      "pr": 2829,
+      "type": "correction",
+      "status": "candidate",
+      "id": 29
```
Comment (jan-ivar) [inline amendments.json:289]:
```suggestion
      "id": 31
```

File: amendments.json [hunk 2]
```diff
@@ -439,6 +446,24 @@
...
+    },
+  ],
+  "replacetrack-reject": [
+    {
+      "description": "Reject setParameters(), replaceTrack(), & insertDTMF() after stop()",
+      "pr": 2829,
+      "type": "correction",
+      "status": "candidate",
+      "id": 31
+  ],
```
Comment (jan-ivar) [inline amendments.json:458]:
```suggestion
    }
  ],
```

**Review by henbos — APPROVED** [review]

File: amendments.json [hunk 3]
```diff
@@ -280,6 +280,13 @@
       "type": "correction",
       "status": "candidate",
       "id": 24
+    },
+    {
+      "description": "Reject setParameters(), replaceTrack(), & insertDTMF() after stop()",
+      "pr": 2829,
+      "type": "correction",
+      "status": "candidate",
+      "id": 31
```
Comment (jan-ivar) [inline amendments.json:289]:
```suggestion
      "id": 32
```

File: amendments.json [hunk 4]
```diff
@@ -439,6 +446,25 @@
...
       "id": 30
+    },
+  ],
+  "replacetrack-reject": [
+    {
+      "description": "Reject setParameters(), replaceTrack(), & insertDTMF() after stop()",
+      "pr": 2829,
+      "type": "correction",
+      "status": "candidate",
+      "id": 31
```
Comment (jan-ivar) [inline amendments.json:457]:
```suggestion
    }
  ],
  "replacetrack-reject": [
    {
      "description": "Reject setParameters(), replaceTrack(), & insertDTMF() after stop()",
      "pr": 2829,
      "type": "correction",
      "status": "candidate",
      "id": 32
```

File: amendments.json [hunk 5]
```diff
@@ -439,6 +446,25 @@
...
+    }
+  ],
+  "dtmf-stopping": [
+    {
+      "description": "Reject setParameters(), replaceTrack(), & insertDTMF() after stop()",
+      "pr": 2829,
+      "difftype": "append",
+      "type": "correction",
+      "status": "candidate",
+      "id": 31
```
Comment (jan-ivar) [inline amendments.json:467]:
```suggestion
      "id": 32
```

**Pull request merged**